consensus/istanbul/backend/handler.go

```diff
@@ -337,10 +332,6 @@ func (sb *Backend) newChainHead(newBlock *types.Block) {
 	// * If this is a node maintaining validator connections (e.g. a proxy or a standalone validator), refresh the validator enode table.
 	// * If this is a proxied validator, notify the proxied validator engine of a new epoch.
 	if istanbul.IsLastBlockOfEpoch(newBlock.Number().Uint64(), sb.config.Epoch) {
-
-		sb.coreMu.RLock()
-		defer sb.coreMu.RUnlock()
-
```

Comment:
Same as above, how can you be sure that this mutex is not required here?

Reply:
Also same as my [response](https://github.com/celo-org/celo-blockchain/pull/1654/files#r681963597) for your above comment